src/main/java/hu/bme/mit/spaceship/TorpedoStore.java

```diff
@@ -30,18 +31,17 @@ public TorpedoStore(int numberOfTorpedos){
 
   public boolean fire(int numberOfTorpedos){
     if(numberOfTorpedos < 1 || numberOfTorpedos > this.torpedoCount){
-      new IllegalArgumentException("numberOfTorpedos");
+      throw new IllegalArgumentException("numberOfTorpedos");
```

Comment:
You should provide more detail on why the provided argument was illegal. Otherwise I like the idea of actually throwing an exception after creating it. Very good (y)